### PR TITLE
feat: use exportEvents

### PR DESCRIPTION
**Pull request by yakkomajuri**
Wrapping up my day and putting this up even though not all test cases are covered.

But with exportEvents tested and setupPlugin not really having changed we should feel comfortable deploying this without spinning up a BQ instance

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
 import { BigQuery, Table, TableField, TableMetadata } from '@google-cloud/bigquery'
 
 type BigQueryPlugin = Plugin<{
     global: {
         bigQueryClient: BigQuery
         bigQueryTable: Table
-        bigQueryTableFields: TableField[]
-
-        exportEventsBuffer: ReturnType<typeof createBuffer>
         exportEventsToIgnore: Set<string>
```
Comment (macobo) [inline index.ts:8]: Meta: A whitelist is generally better than a blacklist.

File: index.ts [hunk 2]
```diff
@@ -146,120 +144,57 @@ export async function exportEventsToBigQuery(events: ProcessedPluginEvent[], { g
...
                     event: eventName,
                     properties: JSON.stringify(ingestedProperties || {}),
-                    elements: JSON.stringify(shouldExportElementsForEvent && elements ? elements : {}),
+                    elements: JSON.stringify(elementsToExport),
                     set: JSON.stringify($set || {}),
                     set_once: JSON.stringify($set_once || {}),
                     distinct_id,
                     team_id,
                     ip,
+                    site_url: '',
```
Comment (macobo) [inline index.ts:170]: Context on this change?  Is there value in always uploading a blank column?

**Review by macobo — APPROVED** [review]
Minor question, otherwise looks great!

File: index.ts (hunk 2 above)
Comment (yakkomajuri) [reply index.ts:170]: Fixes a potential issue I introduced in the previous PR. We stopped sending `site_url` with the event onEvent gets but this plugin was still sending it as undefined.

I removed it [here](https://github.com/PostHog/bigquery-plugin/commit/0fb438afcc850dc3f4f7f65285430110123a1381) but decided to keep sending it as an empty string to be safe. Unsure if BQ will complain about a value missing.

**Pull request merged**